Disable building documentation which requires the dvips utility.
This patch should be dropped once we include a native version of
dvips.

Upstream-Status: Inappropriate [Other]
Temporary workaround which disables documentation.

Signed-off-by: Scott Garman <scott.a.garman@intel.com>

diff -urN linuxdoc-tools-0.9.66.orig/doc/Makedoc.sh linuxdoc-tools-0.9.66/doc/Makedoc.sh
--- linuxdoc-tools-0.9.66.orig/doc/Makedoc.sh	2011-02-25 15:26:41.142917782 -0800
+++ linuxdoc-tools-0.9.66/doc/Makedoc.sh	2011-02-25 15:27:25.141917472 -0800
@@ -52,13 +52,13 @@
 #	$TMPDIR/sgml2latex --pass="\usepackage{times}" -o dvi ./guide
 #fi
 
-if [ -n "`which dvips`" ]; then
-	echo "   + dvips" >&2
-	dvips -t letter -o ./guide.ps ./guide.dvi
-	if [ -n "`which gzip`" -a -f ./guide.ps ]; then
-		gzip -fN ./guide.ps
-	fi
-fi
+#if [ -n "`which dvips`" ]; then
+#	echo "   + dvips" >&2
+#	dvips -t letter -o ./guide.ps ./guide.dvi
+#	if [ -n "`which gzip`" -a -f ./guide.ps ]; then
+#		gzip -fN ./guide.ps
+#	fi
+#fi
 
 
 echo "- Building info docs" >&2
